src/cmd/promremotebench/main.go

```diff
@@ -108,12 +118,24 @@ func main() {
 		queryDebugLength   = flag.Int("query-debug-length", 128, "Query debug character length to print, use zero to print entire response")
 	)
 
+	// Can have multiple write-targets.
+	var writeTargetURLs targetUrls
+	flag.Var(&writeTargetURLs, "target", "Target remote write endpoint (for remote write)")
+
+	// Can have multiple query-targets.
+	var queryTargetURLs targetUrls
+	flag.Var(&queryTargetURLs, "query-target", "Target query endpoint(s) (for exercising by proxy remote read)")
 	flag.Parse()
-	if len(*targetURL) == 0 {
+
+	if len(writeTargetURLs) == 0 {
```

Comment:
The default behaviour before would also set a default target url of `http://localhost:7201/receive` although I don't think it really matters. It's likely that these targets always need to be set.

Reply:
Yeah it was a little weird before that it had default behaviour and then overrode it only if they'd supplied something (but only failed on nil?) I'll just set it to default if empty :+1: